kodilla-advanced-tests/src/main/java/com/kodilla/mockito/homework/WeatherAlertsService.java

```diff
@@ -0,0 +1,48 @@
+package com.kodilla.mockito.homework;
+
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class WeatherAlertsService {
+
+    private Map<String, Set<Subscriber>> weatherAlertService = new HashMap<>();
+
+    public void addSubscriber(String location, Set<Subscriber> subscribers ) {
```

Comment:
nie rozumiem - co ta metoda ma robić? Jej nazwa mówi całkiem coś innego niż to, co w jej ciele

Reply:
dodanie subskrybentów do danej lokalizacji, zmieniłam nazwę metody